features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -656,6 +656,12 @@
     <configfile finalname="${openhab.conf}/services/influxdb.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/influxdb</configfile>
   </feature>
 
+  <feature name="openhab-persistence-gcal" description="GCal Persistence" version="${project.version}">
+    <feature>openhab-runtime-base</feature>
+    <feature>openhab-runtime-compat1x</feature>
```

Comment:
openhab-persistence-gcal depends on openhab-io-gcal1 (it imports the `org.openhab.io.gcal.auth` package)